### ISSUE TITLE
Not able to add subscriptions or holidays calendars

### ISSUE THREAD

**Issue by gio-educ** [nextcloud/calendar#6095]
### Steps to reproduce

1. Go to calendar app
2. Click on "+ New calendar"
3. You won't find the option "New subscription from link (read-only) as described [here](https://docs.nextcloud.com/server/latest/user_manual/en/groupware/calendar.html#create-a-new-calendar)
4. You won't find the option "add holiday calendar" either as described [here](https://docs.nextcloud.com/server/latest/user_manual/en/groupware/calendar.html#subscribe-to-a-holiday-calendar)  
 

### Expected behavior

To find those options as described in the calendar user manual

### Actual behaviour

One see only these 2 options:

- New calendar
-  New calendar with task

![image](https://github.com/nextcloud/calendar/assets/97684858/5e5aca9e-c1aa-467e-ac37-a2acce8294c1)

### Calendar app version

4.7.8

### Web server

None

### Database engine version

None

### PHP engine version

None

### Nextcloud version

28,0,6 and 29.0.3

### Updated from an older installed version or fresh install

Updated from an older version

**Comment by tcitworld** [comment]
Is the `allow_calendar_link_subscriptions` config set by any chance?
What does 
```
occ config:app:get dav allow_calendar_link_subscriptions --details
```
returns?

**Comment by gio-educ** [comment]
2 different outcomes:

- NC 28.0.6
                                          
  The "--details" option does not exist.  
                                          

config:app:get [--output [OUTPUT]] [--default-value [DEFAULT-VALUE]] [--] <app> <name>

- NC 29.0.3:

In AppConfig.php line 203:
                      
  unknown config key  
                      

config:app:get [--output [OUTPUT]] [--details] [--default-value [DEFAULT-VALUE]] [--] <app> <name>

**Comment by gio-educ** [comment]
Fixed in both NC 28.0.6 and 29.0.3 after updating to Calendar app version 4.7.10, thanks!
